storages/clickhouse/clickhouse.go

```diff
@@ -369,7 +316,14 @@ func (ch *clickHouse) tempTableSamples(ctx context.Context, start, end int64, fi
 	return ch.scanSamples(rows)
 }
 
-func (ch *clickHouse) Read(ctx context.Context, queries []base.Query) (*prompb.ReadResponse, error) {
+func (ch *clickHouse) querySamplesNew(ctx context.Context, start, end int64, fingerprints map[uint64]struct{}) {
```

Comment:
what does this function do ?

Reply:
Absolutely nothing. A left-over. Good find